### PR TITLE
add user avatar to application page

### PR DESCRIPTION
**Pull request by wokoci**
Just added the developer avatar to the app page

### CODE REVIEW

File: views/releases-pipeline.html [hunk 1]
```diff
@@ -39,12 +39,14 @@
             <!-- Dashboard component gets rendered here -->
         </div>
     </body>
```
Comment (ckalista) [inline views/releases-pipeline.html:41]: Maybe there is no need to reformat here? Also there are not needed blank spaces...

**Review by ckalista — CHANGES_REQUESTED** [review]

**Comment by ckalista** [comment]
Two additional comments: 
- The avatar should be alined on the lefthand side of the header.
- The avatar size should be such as the overall header width doesn't increase.

Also can you push the newly generated JS artifact. Use the following command to generate it:
`npm run-script build`

File: views/releases-pipeline.html (hunk 1 above)
Comment (wokoci) [reply views/releases-pipeline.html:41]: Code reformated

**Comment by ckalista** [comment]
Code review completed - looks good.

**Review by ckalista — APPROVED** [review]

**Pull request merged**